acquia_lift.admin.wizard.inc

```diff
@@ -1469,6 +1544,109 @@ function _acquia_lift_personalize_campaign_wizard_targeting_audience(&$form, &$f
+  $form['control'] = array(
+    '#type' => 'fieldset',
+    '#tree' => FALSE,
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+    '#title' => t('Control (@control_rate%)', array('@control_rate' => $control_rate)),
+  );
+  $form['control']['control_rate'] = array(
+    '#type' => 'acquia_lift_percentage',
+    '#title' => t('Control Group'),
+    '#field_suffix' => '%',
+    '#size' => 3,
+    '#description' => t('A fixed baseline variation will be shown, by default the first variation in the set.'),
+    '#default_value' => $control_rate,
+    '#rest_title' => t('Test Group'),
+    '#rest_description' => t('Personalized variations will be shown.'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+    '#disabled' => !$editable,
+  );
+  $decision_style = isset($agent_data->data['decision_style']) ? $agent_data->data['decision_style'] : 'adaptive';
+  $form['decision_style'] = array(
+    '#type' => 'radios',
+    '#title' => t('Decision Style'),
+    '#options' => array('adaptive' => t('Auto-personalize'), 'random' => t('Test only')),
+    '#default_value' => $decision_style,
+    '#title_display' => 'invisible',
+    '#disabled' => !$editable,
+  );
+  $form['decision_style']['adaptive'] = array(
+    '#description' => t('Adapts to users and chooses the best option over time.'),
+  );
+  $form['decision_style']['random'] = array(
+    '#description' => t('Tests variations and reports results.'),
+  );
+  $explore_rate = isset($agent_data->data['explore_rate']) ? $agent_data->data['explore_rate'] : 20;
+  $decision_style_parents = $element_parents;
+  $decision_style_parents[] = 'decision_style';
+  $decision_style_form_element = '';
+  foreach ($decision_style_parents as $i => $parent_name) {
+    $decision_style_form_element .= ($i ? '[' . $parent_name . ']' : $parent_name);
+  }
+  $form['distribution'] = array(
+    '#type' => 'fieldset',
+    '#tree' => FALSE,
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+    '#title' => t('Distribution (@explore_rate/@rest)', array(
+      '@explore_rate' => $explore_rate,
+      '@rest' => (100 - $explore_rate),
+    )),
+    '#states' => array(
+      'visible' => array(
+        ':input[name="' . $decision_style_form_element . '"]' => array('value' => 'adaptive'),
+      ),
+    ),
+  );
+  $form['distribution']['explore_rate'] = array(
+    '#type' => 'acquia_lift_percentage',
+    '#title' => t('Random Group'),
+    '#field_suffix' => '%',
+    '#description' => t('Variations will be shown randomly and tracked to adjust for false positives.'),
+    '#size' => 3,
+    '#default_value' => isset($agent_data->data['explore_rate']) ? $agent_data->data['explore_rate'] : 20,
+    '#rest_title' => t('Personalized Group'),
+    '#rest_description' => t('The "best" variation will be shown for each visitor based on our algorithm.'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+    '#disabled' => !$editable,
+  );
```

Comment:
Right - we removed optionsForm and optionsFormValidate.  I took this code, however, from the 7.x-1.x branch where they still exist.